Modelica_LinearSystems2/package.mo

```diff
@@ -1,5 +1,5 @@
 within ;
-package Modelica_LinearSystems2 "Modelica_LinearSystems2 (version 3.0.0) - Analysis, Synthesis and Modeling of Continuous and Discrete Linear Systems"
+package Modelica_LinearSystems2 "Modelica_LinearSystems2 (version 3.0.1) - Analysis, Synthesis and Modeling of Continuous and Discrete Linear Systems"
```

Comment:
I'm a bit irresolute regarding the version number. simver.org says 
> MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes

Changeing dependency on MSL seems being backward compatible. But we have neither bug fix nor new features.
I can live with v3.0.1, just want to point this out.